go.mod

```diff
@@ -1,25 +1,23 @@
-module github.com/lyft/flytectl
+module github.com/flyteorg/flytectl
 
 go 1.13
 
 require (
 	github.com/dustin/go-humanize v1.0.0 // indirect
+	github.com/flyteorg/flyteidl v0.18.15
+	github.com/flyteorg/flytestdlib v0.3.13
 	github.com/ghodss/yaml v1.0.0
-	github.com/golang/protobuf v1.3.2
+	github.com/golang/protobuf v1.4.3
 	github.com/kataras/tablewriter v0.0.0-20180708051242-e063d29b7c23
 	github.com/landoop/tableprinter v0.0.0-20180806200924-8bd8c2576d27
-	github.com/lyft/flyteidl v0.18.11
-	github.com/lyft/flytestdlib v0.3.10-0.20200619054107-45f341b716fa
 	github.com/mattn/go-runewidth v0.0.9 // indirect
-	github.com/mitchellh/mapstructure v1.1.2
-	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/afero v1.2.2 // indirect
-	github.com/spf13/cobra v0.0.5
+	github.com/mitchellh/mapstructure v1.4.1
+	github.com/sirupsen/logrus v1.7.0
+	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.7.0
 	github.com/yalp/jsonpath v0.0.0-20180802001716-5cc68e5049a0
-	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975 // indirect
-	google.golang.org/grpc v1.26.0
-	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/apimachinery v0.18.3 // indirect
+	google.golang.org/grpc v1.35.0
+	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/client-go v11.0.0+incompatible // indirect
```

Comment:
Why do we need k8s client?
And if so, can we update it to - https://github.com/flyteorg/flytepropeller/blob/master/go.mod#L30

I feel we do not need it - lets delete it?

Reply:
Doesn't look like was needed.  Is Deleted.